web_development_101/git_basics/introduction_to_git.md

```diff
@@ -37,12 +37,12 @@ By the end of this lesson, you should be able to do the following:
 </div>
 
 ### Additional Resources
-This section contains helpful links to other content. It isn't required, so consider it supplemental for if you need to dive deeper into something.
```

Comment:
This is the standard additional resources wording for every page. I think it should be kept consistent across the site.

Reply:
That makes sense. I restored the original language.